packages/web-app-files/src/helpers/share/role.ts

```diff
@@ -59,6 +59,10 @@ export abstract class ShareRole {
     })
   }
 
+  public permissionsByBitmask(bitmask: number, allowSharing: boolean): SharePermission[] {
```

Comment:
Can be deleted if `SharePermissions.bitmaskToPermissions` solves your issue from the `RoleDropdown`.